cmd/proxy/cmd.go

```diff
@@ -9,6 +9,9 @@ type ServerCmd struct {
 	PromAddr        string `arg:"--prom-addr" help:"address of prometheus metrics endpoint" default:"0.0.0.0:5004"`
 	PrefetchWorkers int    `arg:"--prefetch-workers" help:"number of workers to prefetch content" default:"50"`
 
+	// Leader election namespace.
+	K8sNamespace string `arg:"--k8s-namespace" help:"namespace for leader election" default:"peerd-ns"`
+
```

Comment:
Instead of explicitly setting the namespace you could read it from `/var/run/secrets/kubernetes.io/serviceaccount/namespace` within the pod. This way you can auto-detect the namespace and you don't need to thread the needle.

Reply:
Nice. Added this.